system-int-test/test/cmr/system_int_test/ingest/ingest_lifecycle_integration_test.clj

```diff
@@ -147,6 +149,9 @@
 (deftest mmt-ingest-round-trip
   (testing "ingest and search UMM JSON metadata"
     ;; test for each UMM JSON version
+  (let [accepted-version (common-config/collection-umm-version)
```

Comment:
I've noticed that this bit of code got added to just about every one of the system-int tests. Is this something that unit tests could replace, or is it necessary for these tests to function?

Reply:
We should always test the latest version, which is ver/current-collection-version. But since ingest limits it to common-(config/collection-umm-version), we need to set it to the latest version in our tests.

Reply:
How about moving this to the fixture and use `use-fixture` to set it rather than putting this in every test?

Reply:
Because there are still some tests there that want to test the config/collection-umm-version, I think.  Daniel created many of the curr-ingest-ver-example-collection-record. I didn't remove them all, I just fixed the few ones that failed for me.